src/libvirtApi/domain.js

```diff
@@ -761,11 +759,13 @@ export function domainSetVCPUSettings ({
     threads,
     isRunning
 }) {
-    return call(connectionName, objPath, 'org.libvirt.Domain', 'GetXMLDesc', [Enum.VIR_DOMAIN_XML_INACTIVE], { timeout, type: 'u' })
-            .then(domXml => {
-                const updatedXML = updateVCPUSettings(domXml[0], count, max, sockets, cores, threads);
-                return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'DomainDefineXML', [updatedXML], { timeout, type: 's' });
-            });
+    const opts = { err: "message", environ: ['LC_ALL=C'] };
```

Comment:
Not necessary for this PR, but in general: C.UTF-8 is a thing everywhere since RHEL 8, and preferable in general.